### DISCUSSION TITLE
高级中怎么修改字体大小？

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#173] [求助 / Q&A]
![image](https://github.com/zotero-chinese/wiki/assets/64388319/8a89ca6b-d594-4610-8791-bcfe0c4266a6)
请问在高级中怎么修改字体大小，我点击没有反应？

谢谢啦

**Comment by jiaojiaodubai** [comment]
我在Windows下更改这一选项，有效

**Reply by LHT-Liu** [reply]
mac下的这个感觉有点问题
